homework jquerry/normal/js/script.js

```diff
@@ -0,0 +1,59 @@
+$(document).ready(function() {
+    $('#one').click(function() {
```

Comment:
Too many click listeners. Why don't use one and depending on this.target make corresponding operation :)